src/test/java/nodejs/flows/users/users.feature

```diff
@@ -0,0 +1,25 @@
+Feature: get all users data
+
+  Background:
+    * url urlNodeJS
+    * def usersPageSchema = read('classpath:nodejs/schemas/users/usersPage.json')
+    * def userSchema = read('classpath:nodejs/schemas/users/users.json')
+
+  Scenario Outline: get user data as regular user
+    * def auth = call read('classpath:nodejs/flows/auth/loginRole.feature@<role>')
+    * def token = auth.token
+
+    Given path 'users'
+    And header Authorization = token
+    When method get
+    Then status 200
+    And match response == usersPageSchema
+    And match each response.page == userSchema
```

Comment:
you are validating the schema of the page before that line but in this one you are assumed the field `page` is always present. The field page belong to the schema of the page.

Reply:
I dont understand what you meant here.
Anyway I made a change to do the assertion in just one line using the karate markers syntax (maybe thats what you meant 😛)